datasets/us/ge/med_exclusions/us_ge_med_exclusions.yaml

```diff
@@ -0,0 +1,35 @@
+title: US Georgia Healthcare provider exclusions
+entry_point: crawler.py
+prefix: us-medge
+coverage:
+  frequency: daily
+  start: "2024-08-15"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of providers who have been terminated for cause from Georgia's Medicaid program.
```

Comment:
```suggestion
  Providers who have been terminated for cause from Georgia's Medicaid program.
```

Picture this as the description under the search result. We don't necessarily want to copy/paste what they've said - google sometimes frowns upon that. But just for a sense of how this is used.

![image](https://github.com/user-attachments/assets/29f1aa54-c64a-4694-9989-22ef5f86bf34)